admin/app/controllers/AnalyticsController.scala

```diff
@@ -1,13 +1,13 @@
 package controllers.admin
 
-import play.api.mvc.{Action, Controller}
+import play.api.mvc.{Action, AnyContent, Controller}
 import common.{ExecutionContexts, Logging}
 import model.{ApplicationContext, NoCache}
 
 import scala.concurrent.Future
 
 class AnalyticsController(implicit context: ApplicationContext) extends Controller with Logging with ExecutionContexts {
-    def abtests() = Action.async { implicit request =>
+    def abtests(): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
For me, this makes the code more difficult to read but if compilation time is an issue it's worthwhile.